### ISSUE TITLE
Underscores don't seem to be accepted in request URL

### ISSUE THREAD

**Issue by ghost** [usablica/liaar#1]
Here is the error returned by the application:

```
[2014-08-05T10:56:14.911008] [info] Processing request for /MOE_v/v1/OrderHistory/profile
Unhandled Error
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/protocols/basic.py", line 564, in dataReceived
    why = self.lineReceived(line)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/http.py", line 1557, in lineReceived
    self.allContentReceived()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/http.py", line 1624, in allContentReceived
    req.requestReceived(command, path, version)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/http.py", line 773, in requestReceived
    self.process()
--- <exception caught here> ---
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/server.py", line 156, in process
    self.render(resrc)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/server.py", line 191, in render
    body = resrc.render(self)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/twisted/web/resource.py", line 216, in render
    return m(request)
  File "build/bdist.macosx-10.9-intel/egg/liaar/lib/twisted_handler.py", line 19, in render_GET

  File "build/bdist.macosx-10.9-intel/egg/liaar/lib/exception.py", line 13, in handle

exceptions.Exception: No route found for: /MOE_v/v1/OrderHistory/profile
```

**Comment by afshinm** [comment]
Ops, I think I should modify the regex. Thanks for report.

**Comment by afshinm** [comment]
@bolchisbolchis Solved, please use v0.2.0.
